lib/utilities.js

```diff
@@ -4,6 +4,24 @@ import _ from 'lodash';
 import { log } from './logger';
 import { findDeveloperImage } from './imagemounter/utils/list_developer_image';
 
+/**
+ * Converts bytes to a human-readable size string
+ * @param {number} bytes - Number of bytes to convert
+ * @param {number} [decimals=2] - Number of decimal places
+ * @returns {string} Human-readable size string (e.g., "1.23 MB", "456.78 KB")
+ */
+function toReadableSizeString(bytes, decimals = 2) {
```

Comment:
I don't see a need to reimplement this method simply import it as is from the support package 
```
import {util } from '@appium/support';

....

console.log(util.toReadableSizeString(sizeBytes));
...
```